backbone.js

```diff
@@ -1436,8 +1436,7 @@
 
     // Are we at the app root?
     atRoot: function() {
-      var path = this.location.pathname.replace(/[^\/]$/, '$&/');
-      return path === this.root && !this.getSearch();
+      return this.location.pathname === this.root && !this.getSearch();
```

Comment:
I think we'd still need to normalize here because I don't think the result should change based on trailing slash preference.